lib/winrm/http/transport.rb

```diff
@@ -173,13 +253,13 @@ def send_kerberos_request(message)
             'boundary="Encrypted Boundary"'
         }
 
-        body = <<-EOF
---Encrypted Boundary\r
-Content-Type: application/HTTP-Kerberos-session-encrypted\r
-OriginalContent: type=application/soap+xml;charset=UTF-8;Length=#{original_length + pad_len}\r
---Encrypted Boundary\r
-Content-Type: application/octet-stream\r
-#{emsg}--Encrypted Boundary\r
+        body = <<-EOF.gsub(/^\s{10}/,"")
```

Comment:
This was to clean up the format indentation in the 'rb' file. It will remove the leading spaces caused by the indentation. It's always been such an eye-sore scanning that file and having the HEREDOC not indented.